packages/services/f-analytics/src/plugins/lib/analytics.service.js

```diff
@@ -0,0 +1,12 @@
+export default class AnalyticService {
+    constructor (store, options) {
+        this.namespace = options.namespace;
+        this.store = store;
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    push (event) {
+        // TODO; // eslint-disable-line
```

Comment:
What error comes back here?

Reply:
No error it just returns what you posted; the event.